state/vote.go

```diff
@@ -2,16 +2,44 @@ package state
 
 import (
 	"fmt"
+	"math/big"
 
+	"github.com/vocdoni/arbo"
+	"github.com/vocdoni/vocdoni-z-sandbox/crypto/ecc/curves"
 	"github.com/vocdoni/vocdoni-z-sandbox/crypto/elgamal"
 )
 
 // Vote describes a vote with homomorphic ballot
 type Vote struct {
-	nullifier     []byte
-	elgamalBallot *elgamal.Ciphertext
-	// address       []byte
-	// commitment    big.Int
+	Nullifier  []byte
+	Ballot     *elgamal.Ciphertext
+	Address    []byte
+	Commitment big.Int
+}
+
+// NewVote creates a new vote
+func NewVote(nullifier, amount uint64) Vote {
```

Comment:
This looks like a test function, if so please rename it to NewVoteForTest()

Reply:
good catch, definitely only used for testing so i actually moved this to statetransition_test instead